src/server/lib/templateRules.json

```diff
@@ -3442,8 +3471,8 @@
         "2.1"
       ]
     },
-    "Number_Workers_Completing_Sectoral__c": {
-      "key": "Number_Workers_Completing_Sectoral__c",
+    "Number_Workers_Competing_Sectoral__c": {
```

Comment:
@jcomeau1 yup this is pretty maddening

Reply:
Yeah, it drives me nuts. But I bet they'll fix it the day before the period closes :)